EazyE2E/Element/EzElement.cs

```diff
@@ -51,11 +60,17 @@ public void BringIntoFocus()
             this.BackingAutomationElement.SetFocus();
         }
 
-        public EzElement FindChildByName(string name, SearchType type = SearchType.Children)
+        public EzElement FindChildByName(string name, bool searchRaw = false)
         {
-            return new EzElement(_backingAutomationElement.FindFirst(TreeScope.Children, new PropertyCondition(AutomationElement.NameProperty, name)));
-        }
+            if (!searchRaw) return new EzElement(_backingAutomationElement.FindFirst(TreeScope.Children, new PropertyCondition(AutomationElement.NameProperty, name)));
 
+            var walker = TreeWalker.RawViewWalker;
+            var firstChild = walker.GetFirstChild(_backingAutomationElement);
+            if (firstChild.Current.Name == name) return new EzElement(firstChild);
+
+            return null;
```

Comment:
This should not have been committed.  Since we have no users I guess it's not a big deal right now.  We just need to find a way at some point to walk the tree using a raw view.